group_vars/all

```diff
@@ -62,11 +67,19 @@ galaxy_config:
     ftp_upload_dir: "{{ proftpd_files_dir }}"
     allow_user_dataset_purge: True
     allow_user_impersonation: True
+    tool_sheds_config_file: "{{ galaxy_config_dir }}/tool_sheds_conf.xml"
     static_enabled: False
     nginx_upload_store: /tmp
     use_pbkdf2: "{{ use_pbkdf2 }}"
     len_file_path: "{{ galaxy_config_dir }}/len"
     filter-with: proxy-prefix
+    # Conda settings
```

Comment:
So all the conda variables here AND the dependency_resolvers_config_file variable can be removed for group_vars/all, right ?